src/config/index.ts

```diff
@@ -40,7 +40,7 @@ export const DYNAMO_TABLE_NAME = getEnvironmentValue('DYNAMO_TABLE_NAME');
 
 // Template
 export const START = 'start';
-export const HOME = 'home';
+export const GITHUB_HOME = 'github-home';
```

Comment:
Note: all `github` page will be moved to the related directory. Can not be done right now as it will break a few features of the app ...